models/clm/shell_scripts/cesm2_2/simple.csh

```diff
@@ -0,0 +1,114 @@
+#!/bin/csh
+#
+# DART software - Copyright UCAR. This open source software is provided
+# by UCAR, "as is", without charge, subject to all terms of use at
+# http://www.image.ucar.edu/DAReS/DART/DART_download
+
+# This file is just to see if CESM can be bullt on whatever architecture.
+# Single instance ... nothing fancy ... no DART ... just something simple.
+
+/glade/work/thoar/CESM/ctsm/cime/scripts/query_config  --compsets clm
```

Comment:
i'm confused about the intent for this script.  if for a user, is it worth trying to use $USER or setting some HOMEDIR/WORKDIR variables?  this has "thoar" in several pathnames and TJH prints other places so it sounds like your own script, but it's in the pull request.  i guess my request here is to either 1) add something to the comments saying this would need to be updated to be used by someone else or 2) make it a bit more portable if that's feasible.

Reply:
I could definitely put in a comment indicating the the 'query_config' command has to come from their CESM clone.
I'll get rid of the TJH bits as well.